.vale.ini

```diff
@@ -0,0 +1,12 @@
+StylesPath = .vale/styles
+
+Vocab = docs
+
+Packages = Hugo, Microsoft
+
+[*.md]
+MinAlertLevel = suggestion
+BasedOnStyles = Vale, Microsoft
+
+# Microsoft.We disallows the use of "we" or "us". We are using this in many places for now.
+Microsoft.We = NO
```

Comment:
Well, in the future this should indeed go away. We mix "we"/"us" vs. "you" way too much in our tutorials. Instead, it should be "Giant Swarm [platform]" (instead of "we") vs. "you"/"customer".

Reply:
I wanted to wait with that one, since I had mixed feelings and wanted some debate before enabling it. I always liked the personal touch of "we" (Giant Swarm) and "you" (the customer). But I'm not sure it comes across as professional, and also I would like to figure out what the alternatives would be.